src/xcode/ENA/ENA/Resources/Localization/de.lproj/usage.html

```diff
@@ -373,7 +373,7 @@ <h4>Einstellungen im Endgerät</h4>
 				<li>Kamerafunktion</li>
 			</ul>
 			<p>Bitte prüfen Sie in den Einstellungen ihres Endgeräts, ob diese Funktionen aktiviert und für die Verwendung der App freigegeben sind.</p>
-			<p>Eine ausführliche Anleitung zur Einrichtung der App unter iOS und Android finden Sie unter [Link]. Die Anleitung dient lediglich der Erläuterung und ist nicht Teil dieser Nutzungsbedingungen.</p>
+			<p>Eine ausführliche Anleitung zur Einrichtung der App unter iOS und Android finden Sie unter [<a>https://www.coronawarn.app/de/</a>]. Die Anleitung dient lediglich der Erläuterung und ist nicht Teil dieser Nutzungsbedingungen.</p>

```

Comment:
I'd remove the braces if it was me ... ;-)

Reply:
Fair point!